### PR TITLE
initial code for signup pages

### PR DESCRIPTION
**Pull request by rushikesh-ladke**

### CODE REVIEW

File: components/ForgotPassword/ForgotPassword_API.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (gaurav7557) [inline components/ForgotPassword/ForgotPassword_API.tsx]: what is the purpose for this one?

File: components/Login/Login.module.scss [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    width: 70%;
+    height: 50%;
+  }
+  
+  #content{
+    background-color: rgb(255, 255, 255);
+    display: flex;
+    flex-direction: column;
+    /* grid-template-rows: 0fr 0fr 0fr 0.5fr 0.8fr 0.8fr 0.7fr 0.8fr 0.3fr; */
+    padding: 80px 20px 30px 50px;
```
Comment (gaurav7557) [inline components/Login/Login.module.scss:32]: use relative padding in stead of px

File: components/Login/Login.module.scss [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    .container {
+      grid-template-columns: 1fr;
+      grid-template-rows: 1fr 5fr;
+      grid-template-areas:
+        "logo"
+        "content"
+    }
+    .login_content {
+      width: 100%;
+      // display: flex;
```
Comment (gaurav7557) [inline components/Login/Login.module.scss]: remove commented code

File: components/ResetPassword/ResetPassword.module.scss [hunk 4]
```diff
@@ -0,0 +1,142 @@
+/* :root {
```
Comment (gaurav7557) [inline components/ResetPassword/ResetPassword.module.scss]: remove

File: components/Signup/Signup.module.scss [hunk 5]
```diff
@@ -0,0 +1,140 @@
+/* :root {
+  --main-radius: 5px;
```
Comment (gaurav7557) [inline components/Signup/Signup.module.scss]: I see this as repeated, can we reuse this for all other cases

File: components/Login/Login.module.scss (hunk 2 above)
Comment (rushikesh-ladke) [reply components/Login/Login.module.scss:32]: I think this is required.

**Pull request merged**